test/DocumentFormat.OpenXml.Tests/Linq/OpenXmlPartTests.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Wordprocessing;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using Xunit;
+
+#nullable enable
```

Comment:
Should we enable nullable on tests? That probably would be nice to verify things...

Reply:
If the assembly under test has nullability enabled, it would make sense to do that for the corresponding unit tests, too. This way, you would more likely identify cases where nullability is incorrect. For example, see my other issue on nullabity of the MainDocunentPart.Document property, for example.

Reply:
Created an issue to track that